src/Humanizer/Frequency.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Humanizer.Localisation;
+
+namespace Humanizer
+{
+    public class Frequency
```

Comment:
This class exposes a big public interface but doesn't have any unit test coverage. Every public method should be extensively unit tested.